src/containers/FormikForm/IngressField.jsx

```diff
@@ -40,35 +39,46 @@ const IngressField = ({
   preview = false,
   concept = false,
   onBlur,
-}) => (
-  <StyledFormContainer>
-    <FormikField
-      noBorder
-      label={t('form.introduction.label')}
-      name={name}
-      showMaxLength
-      maxLength={maxLength}>
-      {({ field }) =>
-        preview ? (
-          <p className="article_introduction">
-            {parse(renderMarkdown(Plain.serialize(field.value), concept))}
-          </p>
-        ) : (
-          <PlainTextEditor
-            id={field.name}
-            {...field}
-            placeholder={placeholder || t('form.introduction.label')}
-            className="article_introduction"
-            data-cy="learning-resource-ingress"
-            handleSubmit={handleSubmit}
-            onBlur={onBlur}
-            plugins={plugins}
-          />
-        )
-      }
-    </FormikField>
-  </StyledFormContainer>
-);
+}) => {
+  const ref = React.useRef(handleSubmit);
```

Comment:
Er ikke helt fan av å kalle ref's for bare `ref`, er litt ryddigere om den kalles for `handleSubmitRef` eller noe i den duren.  Så er det lettere å se hva som skjer når vi ser på koden om 3 år.